### PR TITLE
Updated README with new testnet docs

### PR DESCRIPTION
**Pull request by fedekunze**
Update testnet with new `gaiad` and `gaiacli` commands

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,67 +68,186 @@ You should see:
...
+```
 
-Replace the genesis.json and config.toml files:
+You can find the corresponding genesis files [here](https://github.com/tendermint/testnets). Then replace the `genesis.json`and `config.toml` files:
 
 ```
 rm $HOME/.gaiad/config/genesis.json $HOME/.gaiad/config/config.toml $HOME/.gaiad/config/addrbook.json
 
-wget -O $HOME/.gaiad/config/genesis.json https://raw.githubusercontent.com/tendermint/testnets/master/gaia-4000/genesis.json
+wget -O $HOME/.gaiad/config/genesis.json https://raw.githubusercontent.com/tendermint/testnets/master/gaia-4000/gaia/genesis.json
```
[the remark this answers is not in the document]
Comment (adrianbrink) [reply README.md]: This is wrong. There is no gaia subfolder.

@fedekunze Could you move those genesis files into this repo and update the links?

[the remark this answers is not in the document]
Comment (fedekunze) [reply README.md]: @adrianbrink only the ones related to `gaia-4000` or all of them ?

[the remark this answers is not in the document]
Comment (adrianbrink) [reply README.md]: Only the ones related to gaia-4000.

[the remark this answers is not in the document]
Comment (fedekunze) [reply README.md]: maybe it's better if we only move the files from the latest testnet in order to reduce misunderstanding

[the remark this answers is not in the document]
Comment (fedekunze) [reply README.md]: @adrianbrink done

**Comment by zmanian** [comment]
LGTM

**Pull request merged**